src/main/java/web/cinema/controllers/ShoppingCartController.java

```diff
@@ -33,14 +35,17 @@ public ShoppingCartController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/add-movie-session")
-    public void addMovieSession(@RequestBody ShoppingCartRequestDto scrd) {
+    public void addMovieSession(@RequestBody @Valid ShoppingCartRequestDto scrd,
```

Comment:
```suggestion
    public void addMovieSession(@RequestBody @Valid ShoppingCartRequestDto cartDto,
```
давай хоча б так щоб було зрозуміліше